addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,735 @@
+        fiscal_position_id, invoice_user_id, state, invoice_payment_state,
+        invoice_date, invoice_date_due, invoice_payment_ref, invoice_sent,
+        invoice_origin, invoice_payment_term_id, invoice_partner_bank_id,
+        invoice_incoterm_id, invoice_source_email,
+        invoice_partner_display_name, invoice_cash_rounding_id, old_invoice_id,
+        create_uid, create_date, write_uid, write_date)
+        SELECT message_main_attachment_id, access_token,
+        COALESCE(number, move_name, name, '/'), COALESCE(date, date_invoice,
+        write_date), reference, comment, type, journal_id, company_id,
+        currency_id, partner_id, commercial_partner_id, amount_untaxed,
+        amount_tax, amount_total, residual, amount_untaxed_signed,
+        amount_tax_company_signed, amount_total_company_signed,
+        residual_company_signed, fiscal_position_id, user_id,
+        state, 'not_paid', date_invoice, date_due, name, sent, origin,
+        payment_term_id, partner_bank_id, incoterm_id, source_email,
+        vendor_display_name, cash_rounding_id, id, create_uid, create_date,
+        write_uid, write_date
+        FROM account_invoice ai
+        WHERE ai.state in ('draft', 'cancel')
+        RETURNING old_invoice_id, id""",
+    )
+    ids2 = env.cr.fetchall()
+    if ids1 or ids2:
+        _move_model_in_data(
+            env, ids1 + ids2, 'account.invoice', 'account.move')
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_move_line (company_id, account_id,
+        sequence, name, quantity, price_unit, discount,
+        price_subtotal, price_total, currency_id, partner_id, product_uom_id,
+        product_id, analytic_account_id, display_type, is_rounding_line,
+        move_id, old_invoice_line_id, create_uid, create_date, write_uid,
+        write_date)
+        SELECT ail.company_id, ail.account_id, ail.sequence, ail.name,
+        ail.quantity, ail.price_unit, ail.discount, ail.price_subtotal,
+        ail.price_total, ail.currency_id, ail.partner_id, ail.uom_id,
+        ail.product_id, ail.account_analytic_id, ail.display_type,
+        ail.is_rounding_line, COALESCE(ai.move_id, am.id), ail.id,
+        ail.create_uid, ail.create_date, ail.write_uid, ail.write_date
+        FROM account_invoice_line ail
+        LEFT JOIN account_invoice ai ON ail.invoice_id = ai.id
+        LEFT JOIN account_move am ON am.old_invoice_id = ai.id
+        LEFT JOIN account_account aa ON ai.account_id = aa.id
+        WHERE COALESCE(ai.move_id, am.id) IS NOT NULL AND
+            aa.internal_type in ('receivable', 'payable')""",
+    )
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_move_line (company_id, account_id,
+        sequence, name, price_unit, currency_id, tax_base_amount,
+        tax_line_id, analytic_account_id, move_id, old_invoice_tax_id,
+        create_uid, create_date, write_uid, write_date)
```

Comment:
it is needed to add exclude_from_invoice_tab as TRUE here, because it is showing the tax as a line:

![imagen](https://user-images.githubusercontent.com/28590170/94536839-18f97c80-0243-11eb-9eca-808c1f270d8f.png)

Reply:
This code is not definitive. I will push a later version today.

Reply:
Thanks @pedrobaeza I was just making a comment about an issue I found :smile: 

By the way, great work on OpenUpgrade!!!

Reply:
Please try now with the pushed version if you still have the same problem.